.github/ISSUE_TEMPLATE/install_request.md

```diff
@@ -3,7 +3,7 @@ name: Install request
 about: Request an installation of package in the stack on HPC's
 title: '[INSTALL] <title>'
 labels: 'install'
-assignees: 'Hang-Lei-NOAA, kgerheiser, climbfuji'
+assignees: 'Hang-Lei-NOAA, kgerheiser'
```

Comment:
How about Cheyenne? I am still maintaining that one ...

Reply:
Oh, forgot about that.